src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -739,6 +743,48 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
+    /// <summary>
+    /// PackageInfoWriteConverter is used to Serialize graph package dependency to JSON.
+    /// </summary>
+    public class PackageInfoWriteConverter : JsonConverter
```

Comment:
are you imagining being able to deserialize this information will be part of another PR? Another Converter?

Reply:
Added deserialization to this converter